AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java

```diff
@@ -1795,6 +1795,21 @@ public void run() {
         }
     };
 
+    class ReadTextListener implements ReadText.ReadTextListener{
+        public void onDone(){
+            if(ReadText.getmQuestionAnswer()==Sound.SOUNDS_QUESTION){
+                long delay = mWaitAnswerSecond * 1000;
+                if (delay > 0) {
+                    mTimeoutHandler.postDelayed(mShowAnswerTask, delay);
+                }
+            }else if(ReadText.getmQuestionAnswer()==Sound.SOUNDS_ANSWER){
+                long delay = mWaitQuestionSecond * 1000;
+                if (delay > 0) {
+                    mTimeoutHandler.postDelayed(mShowQuestionTask, delay);
+                }
+            }
+        }
+    }
```

Comment:
Just adding whitespace according to our coding standards, logic seems fine does it work for you like this in your testing?

```suggestion
    class ReadTextListener implements ReadText.ReadTextListener {
        public void onDone() {
            if (ReadText.getmQuestionAnswer() == Sound.SOUNDS_QUESTION) {
                long delay = mWaitAnswerSecond * 1000;
                if (delay > 0) {
                    mTimeoutHandler.postDelayed(mShowAnswerTask, delay);
                }
            } else if (ReadText.getmQuestionAnswer() == Sound.SOUNDS_ANSWER) {
                long delay = mWaitQuestionSecond * 1000;
                if (delay > 0) {
                    mTimeoutHandler.postDelayed(mShowQuestionTask, delay);
                }
            }
        }
    }
```